src/OSPSuite.Presentation/Presenters/ContextMenus/ComparisonCommonContextMenuItems.cs

```diff
@@ -17,6 +19,11 @@ public static IMenuBarButton CompareObjectsMenu(IReadOnlyList<IObjectBase> objec
          return CompareObjectsMenu(objectsToCompare, objectsToCompare.AllNames(), context, container);
       }
 
+      public static IMenuBarButton CompareBuildingBlocksMenu(IReadOnlyList<IBuildingBlock> buildingBlocksToCompare, IOSPSuiteExecutionContext context, IContainer container)
+      {
+         return CompareObjectsMenu(buildingBlocksToCompare, buildingBlocksToCompare.Select(x => x.DisplayName).ToList(), context, container);
+      }
```

Comment:
trying hard to understand how this change fixes the Excel export...

Reply:
it all comes from the context menu creation.